src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/jpa/NonAssociation.kt

```diff
@@ -37,7 +37,9 @@ class NonAssociation(
   var restrictionType: RestrictionType,
   var comment: String,
 
-  var authorisedBy: String? = null, // This is stored for legacy reasons - the data is not returned in the API except on the legacy endpoints
```

Comment:
> data is not returned in the API except on the legacy endpoints

this is not true. this authorising/creating user is returned when listing all non-associations for a given prisoner in the "modern" api